app/components/Vault/index.js

```diff
@@ -1346,8 +1346,17 @@ const Vault = (props) => {
         </Footer>
       </Card.Footer>
     );
+
+  let minDecimals = decimals;
+  if (vault.token.decimals > 6) {
+    minDecimals = decimals - 4;
+  } else if (vault.tokens >= 5) {
```

Comment:
what does 'vault.tokens' represent? should it be 'vault.token.decimals'?

also, is '>= 5' correct? should it be '<= 5' ?

Reply:
yeah agree with @xgambitox

Reply:
> what does 'vault.tokens' represent? should it be 'vault.token.decimals'?

Fixed, thanks.

> also, is '>= 5' correct? should it be '<= 5' ?

I've changed it to '>=4', if there is less than 4 decimals I'm assuming dust amount is probably big enough to be worth showing. Let me know if this makes sense.